buildroot-external/package/tailscale/S46tailscaled

```diff
@@ -50,12 +50,48 @@ case "${1}" in
 		# make sure to set tailscale 'up' with the right cmdline args
 		# shellcheck disable=SC2086
 		${TAILSCALE_EXEC} up ${TAILSCALE_UP_ARGS} >/dev/null 2>&1 &
+
+		## adjust the resolv.conf manually because the original way from tailscale return an error see next comment lines
+		# Health check:
+		#     - dns-os: writing to "/etc/resolv.pre-tailscale-backup.conf" in rename of "/etc/resolv.conf": open /etc/resolv.pre-tailscale-backup.conf: read-only file system
+		#     - dns: writing to "/etc/resolv.pre-tailscale-backup.conf" in rename of "/etc/resolv.conf": open /etc/resolv.pre-tailscale-backup.conf: read-only file system
+
+		# fetch the current tailscale domain
+		domain=$(nslookup $(tailscale ip -1) 100.100.100.100 | grep name | sed -e 's/.*name = [^.]*\.//' -e 's/\.$//')
```

Comment:
@jens-maus can you help me.
If I run the line just on the ccu via SSH ... nslookup $(.....) ..... it works.
If this is running in the script I get the error "grep not found" / "sed not found"